website/docs/patterns/table-multi-select/partials/interaction-concepts/interaction-concepts.md

```diff
@@ -0,0 +1,69 @@
+When combining a Multi-select table with filtering, sorting, or pagination, you may encounter complex scenarios regarding how selected results should be persisted across pages and what the user expects when traversing large data sets.
+
+These interaction guidelines are not fully comprehensive. They should instead be considered as conversation starters between design and engineering on how best to optimize the user experience.
+
+!!! Info
+
+If you run into unique scenarios that these guidelines don’t account for, we would love to hear about them through our normal [support](/about/support) channels.
+!!!
+
+## Consistent communication
+
+To help prevent unnecessary editing or deleting and to ensure users are interacting with the results they intend to, it is vital to consistently communicate their selections, or the existence of a selection, **at all times**.
+
+Persisting this communication in the selected count can help to avoid mishaps, but sometimes it might be necessary to confirm with the user prior to making changes that cannot be reversed.
```

Comment:
```suggestion
Persisting this communication in the selected count can help to avoid mishaps, but it may also be necessary to confirm with the user prior to making changes that cannot be reversed.
```